### DISCUSSION TITLE
Release v0.1.11

### DISCUSSION THREAD

**Discussion by alvin-reyes** [application-research/estuary#538] [Announcements]
## What's Changed
* merge v0.1.9 tag to master - 09152022 by @alvin-reyes in https://github.com/application-research/estuary/pull/440
* add workflow action to build swagger clients  by @snissn in https://github.com/application-research/estuary/pull/450
* Add slack badge to README by @gmelodie in https://github.com/application-research/estuary/pull/475
* Update deploy-staging.yml by @jlogelin in https://github.com/application-research/estuary/pull/483
* bug fixes from refactor by @en0ma in https://github.com/application-research/estuary/pull/491
* merge v0.1.10 tag to master - 10262022 by @alvin-reyes in https://github.com/application-research/estuary/pull/490
* Update Dockerfile by @jlogelin in https://github.com/application-research/estuary/pull/494
* fix: only send pin completion message after blocks are completely copied by @en0ma in https://github.com/application-research/estuary/pull/495
* chore: add retry to staging blocks copy by @en0ma in https://github.com/application-research/estuary/pull/496
* fix: nosync false by @alvin-reyes in https://github.com/application-research/estuary/pull/497
* handle new wallet data cap check by @en0ma in https://github.com/application-research/estuary/pull/503
* Absolute paths when rendering gateway dir hrefs by @neelvirdy in https://github.com/application-research/estuary/pull/505
* Swagger/updates/2022.10.28 by @snissn in https://github.com/application-research/estuary/pull/501
* Add TokenHash and Label to AuthToken, allow revoke by hash by @neelvirdy in https://github.com/application-research/estuary/pull/492
* Pin improvements rc by @snissn in https://github.com/application-research/estuary/pull/472
* guard against duplicate Pin Queue by @snissn in https://github.com/application-research/estuary/pull/458
* checkin first version of durable queue pin manager by @snissn in https://github.com/application-research/estuary/pull/459
* Don't panic if wallet balance exists but has nil datacap by @neelvirdy in https://github.com/application-research/estuary/pull/511
* Change some per-provider logs to warn where appropriate by @neelvirdy in https://github.com/application-research/estuary/pull/512
* Update replace pin swagger annotations by @gmelodie in https://github.com/application-research/estuary/pull/513
* docs: Update annotations for Pinning API endpoints by @simonwo in https://github.com/application-research/estuary/pull/514
* Run code cleanup by @neelvirdy in https://github.com/application-research/estuary/pull/519
* Fix typo in README by @cindywu in https://github.com/application-research/estuary/pull/518
* chore: quick changes to readme.md - rebase to `dev` by @alvin-reyes in https://github.com/application-research/estuary/pull/522
* Fix pinmgr flaky test and remove nil send pattern by @neelvirdy in https://github.com/application-research/estuary/pull/528
* upgrade swagger codegen cli to v3 docker container by @snissn in https://github.com/application-research/estuary/pull/525
* Fix collections add content by @gmelodie in https://github.com/application-research/estuary/pull/515
* docs: Add content types to pinning endpoints by @simonwo in https://github.com/application-research/estuary/pull/532
* Update dependencies. Requires go 1.18 and BasicLimiter -> ScalingLimitConfig by @neelvirdy in https://github.com/application-research/estuary/pull/529
* feat: add DealUUID to logs by @jcace in https://github.com/application-research/estuary/pull/534
* update response message with Cakes suggested text by @snissn in https://github.com/application-research/estuary/pull/517
* chore: Rebase ‘dev’ from ‘master’ by @alvin-reyes in https://github.com/application-research/estuary/pull/536
* Added datecreated to the query at the GET /invites endpoint by @kelindi in https://github.com/application-research/estuary/pull/531
* allow setting max deal prices config via cli flag by @en0ma in https://github.com/application-research/estuary/pull/537

## New Contributors
* @simonwo made their first contribution in https://github.com/application-research/estuary/pull/514
* @cindywu made their first contribution in https://github.com/application-research/estuary/pull/518
* @jcace made their first contribution in https://github.com/application-research/estuary/pull/534

**Full Changelog**: https://github.com/application-research/estuary/compare/v0.1.10...v0.1.11

<hr /><em>This discussion was created from the release <a href='https://github.com/application-research/estuary/releases/tag/v0.1.11'>v0.1.11</a>.</em>

**Comment by en0ma** [comment]
@alvin-reyes can you update this to specify that this release requires go1.18
